### PR TITLE
Add support for FBX (Filmbox)

### LINKED ISSUE #604: Request support for FBX (Filmbox) detection
I want file-type to have fbx file support.

here is reference can be used. https://code.blender.org/2013/08/fbx-binary-file-format-specification/

### PR DESCRIPTION
**Pull request by anatawa12**
Resolves #604 

I could not find any good mime type so I use new mime type instead.

The most popular documentation about fbx: https://code.blender.org/2013/08/fbx-binary-file-format-specification/

### CODE REVIEW

**Comment by anatawa12** [comment]
I fogot to add fbx file

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
> Open a pull request with a title like Add support for Format, for example, Add support for PNG.

File: readme.md [hunk 1]
```diff
@@ -359,6 +359,7 @@ Returns a `Set<string>` of supported MIME types.
 - [`f4b`](https://en.wikipedia.org/wiki/Flash_Video) - Audiobook and podcast ISO base media file format used by Adobe Flash Player
 - [`f4p`](https://en.wikipedia.org/wiki/Flash_Video) - ISO base media file format protected by Adobe Access DRM used by Adobe Flash Player
 - [`f4v`](https://en.wikipedia.org/wiki/Flash_Video) - ISO base media file format used by Adobe Flash Player
+- [`fbx`](https://en.wikipedia.org/wiki/FBX) - A model format used by many creation application invented by Autodesk
```
[the remark this answers is not in the document]
Comment (anatawa12) [reply readme.md:362]: Should I add more information or should I make simpler?

Comment (Borewit) [inline readme.md:362]:
```suggestion
- [`fbx`](https://en.wikipedia.org/wiki/FBX) - FBX (Filmbox) is a proprietary file format (.fbx) developed by Kaydara and owned by [Autodesk](https://en.wikipedia.org/wiki/Autodesk). It is used to provide interoperability between digital content creation applications.
```

[the remark this answers is not in the document]
Comment (Borewit) [reply readme.md:362]: Suggestion: https://github.com/sindresorhus/file-type/pull/605#discussion_r1378468646

[the remark this answers is not in the document]
Comment (anatawa12) [reply readme.md]: Thank you for suggestion! I've committed your suggestion

**Pull request merged**